### DISCUSSION TITLE
데이터베이스 스키마를 정하고 있습니다.

### DISCUSSION THREAD

**Discussion by falconlee236** [Fresh-Route/FreshRoute_BE#3] [General]
데이터 베이스 스키마를 정하고 있습니다.
이 Discussion에서는 제가 생각한 스키마를 시간 순서에 맞춰서 끄적거릴 것 같으니까
마음에 드는 스키마 있으면 의견 남겨주세요

**Comment by falconlee236** [comment]
@yymin1022 @seoljh0722 
역간 거리 정보를 받아왔는데 이걸 어떻게 표현을 해야할까
지금 데이터는 그냥 인접 거리를 숫자로 쓰는데 이렇게 저장하는게 맞을지 의견좀
여러분의 스키마가 필요합니다.
![image](https://github.com/Fresh-Route/FreshRoute_BE/assets/52999093/45a7a6df-d0cd-4279-9716-34c9f48596cc)

**Reply by yymin1022** [reply]
각 역에서 상대역까지 거리 구해서 인접리스트로 두는게 알고리즘 돌리기 제일 낫지않을까요?..

**Reply by falconlee236** [reply]
인접리스트로 그냥 알고리즘 짤때처럼 만들게요 ㅠㅠ
